src/TropicalGeometry/hypersurface.jl

```diff
@@ -260,3 +260,72 @@ end
 #     @req has_attribute(TropH,:tropical_semiring_map) "no tropical semiring map cached"
 #     return get_attribute(TropH,:tropical_semiring_map)
 # end
+#
+
+@doc raw"""
+   tropical_prevariety(F::Vector{MPolyRingElem},nu::TropicalSemiringMap)
```

Comment:
```suggestion
    tropical_prevariety(F::Vector{MPolyRingElem},nu::TropicalSemiringMap)
```
This needs to be indented by 4 spaces.